### ISSUE TITLE
FPS of video input

### ISSUE THREAD

**Issue by LeoniekevandenBulk** [facebookresearch/VMZ#55]
I was wondering if it is correct that you do not encode your video input in a specific FPS, but just keep the FPS that comes with each video? Does this mean that the models should work on video input with any FPS (within reasonable bounds of course)?

Thanks!

**Comment by dutran** [comment]
For UCF101 and HMDB the videos are used the same as provided in the dataset because they already trimmed. For Kinetics, videos are trimmed into 10' clips using ffmpeg with a specific FPS (e.g. 15 fps or 30 fps).

**Comment by LeoniekevandenBulk** [comment]
But in the download script for the Kinetics dataset, there seems to be no indication of any FPS encoding in the ffmpeg command: https://github.com/activitynet/ActivityNet/blob/master/Crawler/Kinetics/download.py#L100, or does it happen somewhere else?

**Comment by LeoniekevandenBulk** [comment]
@dutran, let me rephrase my question. I was trying to figure out with which FPS the pretrained models were trained. I want to know this because I am using the R(2+1)D model for classification on a different dataset, but I am not sure if I need to encode the FPS as the exact same as the model was trained on.

**Comment by dutran** [comment]
@LeoniekevandenBulk the models provided in [https://github.com/facebookresearch/VMZ/blob/master/tutorials/models.md] were trained on 15 fps [we will soon update models trained on full 30 fps].
